bin/sensei.js

```diff
@@ -225,6 +226,42 @@ cli(
     })
     .demandCommand(1, 1, "One command must be specified")
     .strict()
+    .middleware([
+      function handleConfig(argv, yargs) {
+        let configFilePath = argv.config;
+        if (!path.isAbsolute(configFilePath)) {
+          configFilePath = path.join(argv.material, configFilePath);
+        }
+        let config = {};
+        try {
+          config = JSON.parse(fs.readFileSync(configFilePath, "utf-8"));
+        } catch (err) {
+          if (
+            yargs.parsed.defaulted?.config == undefined ||
+            fs.existsSync(configFilePath)
+          ) {
+            console.warn(
+              `Unable to read config file '${configFilePath}' (${err.message}), skipping config.`
+            );
+          }
+        }
+        for (const [name, value] of Object.entries(argv)) {
+          const isDefinedOnCli = !(name in yargs.parsed.defaulted);
+          const isDefinedInConfig = name in config;
+          if (isDefinedOnCli || !isDefinedInConfig) {
+            config[name] = value;
+          }
+        }
+        return config;
+      },
+      function defaultsSlug(argv) {
+        argv.slug = argv.slug || path.basename(argv.material);
+        return argv;
+      },
+    ])
+    .parserConfiguration({
+      "strip-dashed": true,
```

Comment:
I'm not familiar with this option. What does it do?

Reply:
https://github.com/yargs/yargs-parser#strip-dashed

It's to avoid the useless `slide-height` and `slide-width` keys in the resulting `argv`